### PR TITLE
Document data migration and Flyway

### PR DESCRIPTION
**Pull request by sorin-stefan-iovita-sap**
### Description:
Document Flyway migration.

### CODE REVIEW

**Review by emmetsap — DISMISSED** [review]

File: docs/PERSISTENCE.md [hunk 1]
```diff
@@ -2,7 +2,23 @@
 
 ## Data Migration / Flyway
 
-Coming soon
+Corona Warn App Server uses [Flyway](https://flywaydb.org) for database version control. Flyway is an open-source
```
Comment (fredrb) [inline docs/PERSISTENCE.md:5]:
```suggestion
Corona Warn App Server uses [Flyway](https://flywaydb.org) for database version control. Flyway is an open source
```

File: docs/PERSISTENCE.md [hunk 2]
```diff
@@ -2,7 +2,23 @@
...
+database migration tool. It updates the database from one version to a next using migrations.
```
Comment (fredrb) [inline docs/PERSISTENCE.md:6]:
```suggestion
database migration tool. It updates the database from one version to the next using migrations.
```

**Review by fredrb — DISMISSED** [review]
Small suggestions

**Review by EugenM-SAP — DISMISSED** [review]

**Comment by sorin-stefan-iovita-sap** [comment]
> Small suggestions
@fredrb, more than welcome! 😄

**Review by ioangut — APPROVED** [review]

**Review by KevponSAP — APPROVED** [review]

**Pull request merged**